test/extended/testdata/gssapi/fedora/base/fedora-base-pod

```diff
@@ -0,0 +1,38 @@
+{
+    "kind": "Pod",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "fedora-gssapi-base",
+        "namespace": "gssapiproxy"
+    },
+    "spec": {
+        "volumes": [
+            {
+                "name": "os",
+                "hostPath" : {
+                    "path": "OS_ROOT"
```

Comment:
But all I needed was the `oadm policy add-scc-to-user anyuid -z default -n gssapiproxy`.